### PR TITLE
feat: news page template containers - EXO-65437 - Meeds-io/MIPs#68

### PR DESCRIPTION
**Pull request by GouadriaHanen**
add new containers for the page template :
-Container simple table
-Container simple row

### CODE REVIEW

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 1]
```diff
@@ -23,55 +23,73 @@
       xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
       xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
   <name></name>
-  <container id="newsPage"
-             template="system:/groovy/portal/webui/container/UIContainer.gtmpl"
-             cssClass="border-box-sizing">
+  <container template='system:/groovy/portal/webui/container/UISimpleTableContainer.gtmpl'>
```
Comment (ahamdi) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: Let's add an id attribute for the container here

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 2]
```diff
@@ -23,55 +23,73 @@
...
     <access-permissions>Everyone</access-permissions>
-    <container id="topNewsPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-      <access-permissions>*:/platform/users</access-permissions>
-    </container>
-    <container id="newsBodyPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+    <container template='system:/groovy/portal/webui/container/UISimpleRowContainer.gtmpl'>
```
Comment (ahamdi) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: Let's add an id attribute for the container here

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 3]
```diff
@@ -23,55 +23,73 @@
...
-          </portlet>
-          <access-permissions>Everyone</access-permissions>
-          <show-info-bar>false</show-info-bar>
-        </portlet-application>
-        <portlet-application>
-          <portlet>
-            <application-ref>news</application-ref>
-            <portlet-ref>NewsListView</portlet-ref>
-          </portlet>
+      <container template='system:/groovy/portal/webui/container/UIContainer.gtmpl'>
```
Comment (ahamdi) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: Is this container needed ?

Comment (GouadriaHanen) [reply webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: removed!

**Comment by ahamdi** [comment]
@GouadriaHanen Could you resolve conflicts here ?

**Comment by GouadriaHanen** [comment]
> @GouadriaHanen Could you resolve conflicts here ?

done!

**Review by ahamdi — APPROVED** [review]

**Pull request merged**